package_build_list_host_darwin.json

```diff
@@ -79,8 +79,8 @@
         "tiff-4.2.0.15-rev3-ios": "package-system/tiff-ios",
         "python-3.10.5-rev2-darwin": "package-system/python/darwin_x64/package",
         "asn1-0.9.27-rev2-ios": "package-system/asn1-ios",
-        "PhysX-4.1.2.29882248-rev5-mac": "package-system/PhysX-mac",
-        "PhysX-4.1.2.29882248-rev5-ios": "package-system/PhysX-ios",
+        "PhysX-5.1.1-rev1-mac": "package-system/PhysX5/temp/PhysX5-mac",
+        "PhysX-5.1.1-rev1-ios": "package-system/PhysX5/temp/PhysX5-ios",
```

Comment:
This is a replacement of PhysX (4)? If so, then the package-system/PhysX folder should be deleted as well. If not, then we should keep the 4.X entries here, especially if there is a legacy bug fix case.

Reply:
I thought about it and since building PhysX 4.x is not trivial at all I rather not completely lose its build scripts for the moment and give the users the ability to generate both PhysX 4 and 5.

Reply:
Yeah, no reason to delete the old package scripts and so on.  they're tiny and keeping them around means able to rebuild the old packages in case we ever need to...

Reply:
I was leaning on keeping both entries (PhysX-4 and PhysX-5)

Reply:
> I was leaning on keeping both entries (PhysX-4 and PhysX-5)

Sure, there is no harm. I'll add them back.

Reply:
done